src/pwncore/routes/ctf/start.py

```diff
@@ -1,20 +1,168 @@
 from __future__ import annotations
 
+from fastapi import Response
+import uuid
+from tortoise.transactions import atomic
+
 from pwncore.routes.ctf import router
+from pwncore.db import Container, CTF
+from pwncore.container import docker_client
+from pwncore.config import DEV_CONFIG
+
+config = DEV_CONFIG
```

Comment:
I didn't intend to export `DEV_CONFIG` can you write this same line in config.py